migrations/env.py

```diff
@@ -0,0 +1,197 @@
+import os
+from logging.config import fileConfig
+
+from alembic import context
+from alembic.script import write_hooks
+from sqlalchemy import engine_from_config
+from sqlalchemy.event import listen
+
+from config import DB_HOST, DB_NAME, DB_PASSWORD, DB_PORT, DB_TYPE, DB_USERNAME
+from pepys_import.core.store import (  # Don't remove, they are necessary for the discovery of changes!
+    postgres_db,
+    sqlite_db,
+)
+from pepys_import.core.store.db_base import BasePostGIS, BaseSpatiaLite
+from pepys_import.utils.data_store_utils import (
+    create_spatialite_tables_for_postgres,
+    create_spatialite_tables_for_sqlite,
+    is_schema_created,
+)
+from pepys_import.utils.geoalchemy_utils import load_spatialite
+
+DIR_PATH = os.path.dirname(__file__)
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+db_type = config.attributes.get("db_type", DB_TYPE)
+
+driver = "sqlite"
```

Comment:
Do we need to set the driver here, if we're then setting it in the if statements below?

Reply:
No, we don't need it. It's removed:
https://github.com/debrief/pepys-import/pull/337/commits/77bfbb14b2fd4282d5498e75ff023337890ce8b2